apps/sertra/next.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
```

Comment:
Should sertra be hooked up with preconstruct, or do we want to keep this separate?

Reply:
I guess it doesn't matter for now if you intent this to be developed in isolation from the rest of the "projects" in this monorepo, as suggested in one of your other comments. If you start doing something like `import { Foo } from '../../../../packages/components/src/components/Foo`, and want to use `import { Foo } from '@thunderstore/components'` instead, it should be set up. Although fixing the imports later on isn't that big of an issue.